{
  "exploitdb": [],
  "itw": [
    {
      "id": "CVE-2013-1629",
      "description": "pip before 1.3 uses HTTP to retrieve packages from the PyPI repository, and does not perform integrity checks on package contents, which allows man-in-the-middle attackers to execute arbitrary code via a crafted response to a \"pip install\" operation.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2013-1629",
      "reportURL": "https://www.reddit.com/r/Python/comments/17rfh7/warning_dont_use_pip_in_an_untrusted_network_a/",
      "timeStamp": "2021-03-15T16:22:00Z"
    }
  ],
  "trickest": [
    {
      "id": "CVE-2013-1629",
      "url": "https://github.com/Threekiii/Awesome-POC",
      "added_at": "2022-02-10T19:32:13Z"
    },
    {
      "id": "CVE-2013-1629",
      "url": "https://github.com/pypa/pip/issues/425",
      "added_at": "2022-02-10T19:32:13Z"
    },
    {
      "id": "CVE-2013-1629",
      "url": "https://www.reddit.com/r/Python/comments/17rfh7/warning_dont_use_pip_in_an_untrusted_network_a/",
      "added_at": "1970-01-01T00:00:00Z"
    }
  ],
  "nomisec": [],
  "nuclei": [],
  "metasploit": [],
  "holloways": [],
  "openpoc": [
    {
      "cve": "CVE-2013-1629",
      "url": "https://github.com/Threekiii/Awesome-POC",
      "added_at": "2022-02-10T19:32:13Z",
      "score": 0.9,
      "template_for": ""
    },
    {
      "cve": "CVE-2013-1629",
      "url": "https://github.com/pypa/pip/issues/425",
      "added_at": "2022-02-10T19:32:13Z",
      "score": 0.4,
      "template_for": ""
    },
    {
      "cve": "CVE-2013-1629",
      "url": "https://www.reddit.com/r/Python/comments/17rfh7/warning_dont_use_pip_in_an_untrusted_network_a/",
      "added_at": "2021-03-15T16:22:00Z",
      "score": 0,
      "template_for": ""
    }
  ]
}
